migrations/sql/V2020.01.02.03.38__now_review_tables.sql

```diff
@@ -0,0 +1,43 @@
+ALTER TABLE now_application ADD COLUMN ready_for_review_date timestamp with time zone;
+ALTER TABLE now_application ADD COLUMN review_closed_on_date timestamp with time zone;
+
+
+CREATE TABLE IF NOT EXISTS now_application_review_type (
+  now_application_review_type_code character varying(3)PRIMARY KEY,
+  description character varying(100) NOT NULL,
```

Comment:
does there need to be a space between varying(3) and Primary?

Reply:
Haven't had any issues, the () probably keeps the parser happy.